src/JsonSchema/Uri/Retrievers/FileGetContents.php

```diff
@@ -33,11 +33,7 @@ public function retrieve($uri)
                 'header' => "Accept: " . Validator::SCHEMA_MEDIA_TYPE
             )));
 
-        set_error_handler(function() use ($uri) {
-            throw new ResourceNotFoundException('JSON schema not found at ' . $uri);
-        });
-        $response = file_get_contents($uri);
-        restore_error_handler();
+        $response = @file_get_contents($uri);
```

Comment:
Why not:

        if (!file_exists($uri)) {
            throw new ResourceNotFoundException('JSON schema not found at ' . $uri);
        }

        $response = file_get_contents($uri);